server/api/src/rsFileChksum.cpp

```diff
@@ -165,7 +165,7 @@ int fileChksum(
 
     rodsLog(
         LOG_DEBUG,
-        "fileChksum :: final_scheme [%s]  chkstr_scheme [%s]  svr_hash_policy [%s]  hash_policy [%s]",
+        "fileChksum :: final_scheme [%s]  chkstr_scheme [%s]  hash_policy [%s]",
```

Comment:
weird that this was just... wrong

Reply:
Yes, I ran into this a while back when I enabled debug logging.